src/filters/H5Zbzip2/Project.toml

```diff
@@ -0,0 +1,12 @@
+name = "H5Zbzip2"
+uuid = "094576f2-1e46-4c84-8e32-c46c042eaaa2"
+version = "0.16.0"
```

Comment:
The idea will be to sync the version with the main HDF5.jl package version?

I don't have a problem with this, but as you probably already know, these 'independent' packages could also just maintain their own version number that get bumped when we modify the subpackages. 

I think the main benefit of the current approach is simplicity in bumping version numbers across all packages. OTOH, if we keep an independent version number I think then that reduces precompilation when we make new releases. 
From judging changes in the original blosc file, changes to the filter packages are likely to be infrequent.

Reply:
Yes. We can only tag the whole repository, so we might as well keep it in sync with the tags.

Reply:
You can tag a sub-package associated with a subdirectory of a repository via `@JuliaRegistrator register subdir=...`. Makie e.g., uses this approach.

Reply:
What I meant is a tag and release from the git perspective.

Reply:
Let's keep all the subpackages at version `0.1.0` and tag them individually. I think the benefit is we avoid precompilation of the subpackages after new HDF5.jl releases, and most of these filter packages won't be changing frequently, so I believe that makes more sense and follows semver, instead of bumping the submodules in instances were  HDF5.jl was changes but the subpackages were untouched. 


For releases I've been using the registrator bot. It looks like it should be straightforward to accommodate subpackages with the registrator from @thchr comment.

Reply:
Well we can tell the registrator bot whatever version we want, but the question is how to do you mark these versions in Git? Do you plan on making an individual tag for each sub-package?

Reply:
Ah, I see your point now, since these are not git submodules...

Reply:
Thinking about this again, does the git tag version even matter? No. 

So, what'll happen is that the  git tag version will be synced with the HDF5.jl version as is currently the case.
The git tag version will not agree with the tagged version of the filter subpackages, which will have their own independent versions as specified in their own Project.toml. This is the case since we aren't using git submodules, which is more trouble than it's worth.  

To me this makes sense and is desired behavior since we won't have to tag all the filter packages superfluously every time a change is made in the main HDF5.jl module that doesn't modify the filter submodules.